### DISCUSSION TITLE
`强制固定顶栏和侧栏` 的固定顶栏是不是失效了

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1436] [提问 / Q&A] [answered]
`强制固定顶栏和侧栏` 的顶栏指的是上面的按 UP 主／动态类型筛选栏和发动态框？
说实话 [文档相关描述](https://github.com/the1812/Bilibili-Evolved/blob/master/features.md#动态) 没怎么看懂（弱

**Comment by the1812** [comment, marked as answer]
指的是自定义顶栏, 不过好像也应该对原版顶栏起效才对?

**Reply by Geeyun-JY3** [reply]
试了试原版顶栏无效诶

**Reply by the1812** [reply]
对啊 现在是无效的 因为一开始就只针对自定义顶栏写的

**Reply by Geeyun-JY3** [reply]
那行吧。
